dnanexus/src/eggd_athena.sh

```
@@ -67,16 +74,18 @@ main() {
     _set_bool_inputs
     _set_string_inputs
 
-    python3 athena/athena.py \
+    python3 athena/athena.py report \
         --regions "$regions_path" \
```

Shellcheck (0.10.0)

[warning] regions_path is referenced but not assigned.

(SC2154)